api/routes/openAPI/helpers.js

```diff
@@ -36,16 +36,29 @@ module.exports.schema = {
   })
 };
 
-module.exports.requiresAuth = openAPI => {
+module.exports.requiresAuth = (
+  openAPI,
+  { has401 = true, has403 = true } = {}
+) => {
   const authed = { ...openAPI };
   Object.keys(authed).forEach(route => {
     Object.keys(authed[route]).forEach(verb => {
-      authed[route][verb].security = ['sessionCookie'];
+      if (has403) {
+        authed[route][verb].security = ['sessionCookie'];
+      }
+
+      const authResponses = { ...module.exports.responses.unauthed };
+      if (!has401) {
+        authResponses[401] = undefined;
+      }
+      if (!has403) {
+        authResponses[403] = undefined;
+      }
 
       const responses = authed[route][verb].responses;
       authed[route][verb].responses = {
-        ...responses,
-        ...module.exports.responses.unauthed
+        ...authResponses,
+        ...responses
```

Comment:
Reverse the order of these, so if the caller has already set a 401 or 403 status, it will override whatever this method would set.